aries-site/src/pages/templates/index.js

```diff
@@ -1,41 +1,59 @@
 import React from 'react';
+import Link from 'next/link';
+import { Box, Heading, Paragraph } from 'grommet';
 
-import { Layout, NavPage } from '../../layouts';
-import { ComingSoon, DescriptiveHeader, Meta } from '../../components';
-import { getPageDetails } from '../../utils';
+import { CardGrid, ContentCard, Meta } from '../../components';
+import { structure } from '../../data';
+import { Layout, PageIntro } from '../../layouts';
+import { getPageDetails, getParentPage, nameToPath } from '../../utils';
 
 const title = 'Templates';
-const {
-  color,
-  description,
-  icon,
-  pages,
-  name,
-  seoDescription,
-} = getPageDetails(title);
+const pageDetails = getPageDetails(title);
 
-const Templates = () => {
-  const descriptiveHeader = (
-    <DescriptiveHeader
-      background={color}
-      subText={description}
-      icon={icon}
-      title={title}
-    />
-  );
+const cards = structure
+  .map(obj => {
+    const page = obj;
+    const parent = getParentPage(page.name);
+    page.parent = parent;
+    return page;
+  })
+  .filter(page => page.parent !== undefined)
+  .filter(page => page.parent.name === title);
 
+const Templates = () => {
   return (
-    <Layout descriptiveHeader={descriptiveHeader} title={title} isNavPage>
+    <Layout title={title} isLanding>
       <Meta
         title={title}
-        description={seoDescription}
+        description={pageDetails.seoDescription}
         canonicalUrl="https://design-system.hpe.design/templates"
       />
-      {pages.length ? (
-        <NavPage items={pages} topic={name.toLowerCase()} />
-      ) : (
-        <ComingSoon />
-      )}
+      <Box gap="large">
+        <PageIntro
+          image={{
+            src: '/card-templates.png',
+            alt: 'Card deck of HPE Design System template cards',
+            fit: 'cover',
+          }}
+        >
+          <Box justify="center" fill>
```

Comment:
Feels like this Box/Paragraph combo gets reused on each NavPage -- would it be any benefit for it to be its own component? -- something where it would take props `title={title}` `description={pageDetails.description}`

Reply:
Yes it would. Just ran out of time for to be in time for folks to review and review the site.